controllers/volumereplicationgroup_controller.go

```diff
@@ -1776,6 +1773,26 @@ func (v *VRGInstance) updateVRGConditions() {
 	setVRGDataErrorCondition(&v.instance.Status.Conditions, v.instance.Generation, msg)
 }
 
+func (v *VRGInstance) vrgReadyStatus() {
+	if v.instance.Spec.ReplicationState == ramendrv1alpha1.Secondary {
+		v.log.Info("Marking VRG ready with replicating reason")
+
+		msg := "PVCs in the VolumeReplicationGroup group are replicating"
+		setVRGDataReplicatingCondition(&v.instance.Status.Conditions, v.instance.Generation, msg)
+
+		return
+	}
+
+	if v.instance.Spec.ReplicationState == ramendrv1alpha1.Primary {
```

Comment:
This check will be good idea if we set the condition to unknown state outside the 'if` case; otherwise, should we remove this check?

Reply:
ACK. Removing this if condition.